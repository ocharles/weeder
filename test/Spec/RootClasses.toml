roots = []

type-class-roots = false

root-classes = [ "Show", "Ord", "Bar" ]
