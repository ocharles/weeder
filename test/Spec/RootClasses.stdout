main: test/Spec/RootClasses/RootClasses.hs:5:34: (Instance) :: Enum T
